src/server/svr_func.c

```diff
@@ -4621,6 +4623,8 @@ free_prov_vnode(struct pbsnode * pnode)
  * @retval	-3	:	nodes are all done prov, but curr_aoe, does not
  *					match req_aoe
  * @retval	-4	:	left over provisioning just returned
+ * @retval	-5	: job is not eligible to run since it is unconfirmed
```

Comment:
Riyaz can we combine -1 and -5 to just -1, since both represent that node still provisioning.